projects/bp-gallery/src/components/views/picture/sidebar/picture-info/PictureInfo.tsx

```diff
@@ -128,18 +123,24 @@ const PictureInfo = ({
           onTouch={() => setAnyFieldTouched(true)}
         />
       </PictureInfoField>
-      <PictureInfoField title={t('pictureFields.people')} icon={<Person />} type='person'>
-        <TagSelectionField
-          type={TagType.PERSON}
-          tags={picture.person_tags ?? []}
-          allTags={allPeople ?? []}
-          onChange={people => {
-            savePictureInfo({ person_tags: people });
-          }}
-          noContentText={t('pictureFields.noPeople')}
-          createMutation={newPersonTagMutation}
-        />
-      </PictureInfoField>
+      <FaceTaggingUI
+        tags={picture.person_tags ?? []}
+        allTags={allPeople ?? []}
+        onChange={people => {
+          savePictureInfo({ person_tags: people });
+          /*unfortunately I did not find a way to get the id of a person tag that is being deleted, so i had to go
```

Comment:
wenn das nicht ganz ideal ist, vielleicht in eine Funktion auslagern, die dann explizit angefasst werden muss, wenn dieser Aspekt verbessert werden muss

Reply:
Ich habe #435 geöffnet, das wird also später wahrscheinlich sowieso nochmal gefixt :)